src/zeroD/Reactor.cpp

```diff
@@ -592,4 +603,26 @@ void Reactor::setAdvanceLimit(const string& nm, const double limit)
     }
 }
 
+void Reactor:: buildWallJacobian(vector<Eigen::Triplet<double>>& jacVector)
```

Comment:
Minor formatting error:
```suggestion
void Reactor::buildWallJacobian(vector<Eigen::Triplet<double>>& jacVector)
```